chart/templates/webserver/webserver-ingress.yaml

```diff
@@ -31,7 +31,7 @@ metadata:
     heritage: {{ .Release.Service }}
   {{- with .Values.ingress.web.annotations }}
   annotations:
-  {{ toYaml . | indent 4 }}
+{{ toYaml . | indent 4}}
```

Comment:
This was a bug. When using `toYaml` and `indent` functions, there shouldn't be any whitespace in the line.